test/index.test.js

```diff
@@ -0,0 +1,63 @@
+const MemoryDatabase = require('../lib/index');
+
+describe('memory database', () => {
+  let db = null;
+  beforeEach(() => {
+    db = new MemoryDatabase();
+  });
+
+  it('creates an object in the databse', () => {
+    const cat = { name: 'fluffy' };
+    const createdCat = db.create(cat);
+    expect(createdCat.name).toEqual('fluffy');
+  });
+
+  it('can find an object by id', () => {
+    const cat = { _id: 123, name: 'fluffy' };
+    const createdCat = db.create(cat);
+    const foundCat = db.findById(createdCat._id);
+    expect(foundCat).toEqual(createdCat);
+  });
+
+  it('throws an error when trying to find an object that does not exist', () => {
+    expect(() => {
+      db.findById('notARealId');
+    }).toThrowError('No object with _id notARealId');
+  });
+
+  it('can return every object in the store', () => {
+    const obj1 = { name: 'fifi' };
+    const obj2 = { name: 'ralph' };
+    db.create(obj1);
+    db.create(obj2);
+    const foundObjs = db.find();
+    expect(foundObjs[0].name).toEqual('fifi');
```

Comment:
```suggestion
    expect(foundObjs).toEqual([obj1, obj2]);
```
Here is another way to write the two expect statements in this test.